contrib/vstudio/vc15/zlib.rc

```diff
@@ -0,0 +1,32 @@
+#include <windows.h>
+
+#define IDR_VERSION1  1
+IDR_VERSION1	VERSIONINFO	MOVEABLE IMPURE LOADONCALL DISCARDABLE
+  FILEVERSION	 1, 2, 11, 0
+  PRODUCTVERSION 1, 2, 11, 0
```

Comment:
Does it matter that these don't match the version number?  Will upgrades work?

Reply:
IIRC, these version numbers are for zlib, the version numbers I upgraded are for the nuget package only.

Reply:
I was concerned that if this repo was making a `dll` we would fail to replace it with our installer if the version didn't actually change on the file.  However, it looks like this repo just makes static `.lib`s so I think we're ok